### ISSUE TITLE
Button:GetHitRectInsets() returns values in the wrong order

### ISSUE THREAD

**Issue by seblindfors** [Stanzilla/WoWUIBugs#304]
Reproducible example:
```lua
local button = CreateFrame('Button', 'TestButton', UIParent, 'ActionButtonTemplate')

button:SetPoint('CENTER')
button:SetSize(100, 100)

button:SetHitRectInsets(1, 2, 3, 4)
print(button:GetHitRectInsets())
-- 1, 2, 4, 3 (top and bottom switched)
```

**Comment by LenweSaralonde** [comment]
Could be related to #278

**Comment by seblindfors** [comment]
@LenweSaralonde I guess they corrected the Set method, but not this one

**Comment by Meorawr** [comment]
```
/run B = CreateFrame("Button", nil, UIParent, "ActionButtonTemplate") B:SetPoint("CENTER") B:SetSize(100, 100) B:SetHitRectInsets(1, 2, 3, 4) print(B:GetHitRectInsets())
```
Shortening the test case to a one-liner for my future self. Issue still exists.

**Comment by Meorawr** [comment]
Fixed in 10.0.5.47660. Was probably fixed in an earlier build but I hadn't re-tested this specific issue since asking for it to get fixed prior to the holidays.

For Wrath Classic it seems 3.4.1 went live with the fix also applied.
